src/Compilers/CSharp/Test/Semantic/Semantics/RecordTests.cs

```diff
@@ -109,16 +109,35 @@ public void RecordProperties_01()
 using System;
 data class C(int X, int Y)
 {
+    int Z = 123;
     public static void Main()
     {
         var c = new C(1, 2);
         Console.WriteLine(c.X);
         Console.WriteLine(c.Y);
```

Comment:
>Console.WriteLine(c.Y); [](start = 8, length = 23)

nit: Consider printing `c.Z` as well.

Reply:
> nit: Consider printing c.Z as well.

I will add this in the next PR. For now, it is clear in IL that this field is also initialized.

---
In reply to: [434399596](https://github.com/dotnet/roslyn/pull/44803#discussion_r434399596) [](ancestors = 434399596)